backend/lib/routes/info.js

```diff
@@ -51,3 +51,23 @@ async function fetchGardenerVersion () {
     logger.warn(`Could not fetch gardener version. Error: ${err.message}`)
   }
 }
+
+router.route('/identity')
```

Comment:
Since this is static information we should merge it into the frontend `/config.json`.